drakrun/drakrun/storage.py

```diff
@@ -307,6 +330,12 @@ def export_vm0(self, path: str):
     def import_vm0(self, path: str):
         shutil.copy(path, os.path.join(VOLUME_DIR, 'vm-0.img'))
 
+    def delete_vm_volume(self, vm_id: str):
+        # unmount can be done here
```

Comment:
Not really, volume shouldn't be mounted at this point

Reply:
Just to be safe, coz if deletion is occurring, the volume should be unmounted